### ISSUE TITLE
Patella alta

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7333]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Patella alta is a patella that rides abnormally high in relation to the femur, the femoral trochlea, or the tibia, with decreased bony stability requiring increased knee flexion angles to engage the trochlea. 
Comment:
An abnormally high patella may therefore insufficiently engage the proximal trochlea groove both in extension and in the early phase of knee flexion - making it one of the potential risk factors for patellar instability. Accordingly, patella alta is present in 30% of patients with recurrent patellar dislocation.
PMID:29309446

**Parent term (use hpo.jax.org/app)**
Abnormal patella morphology HP:0003045

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
